charts/librespeed/README.md

```diff
@@ -0,0 +1,96 @@
+# Librespeed
+
+HTML5 based speedtest with password protected history
+
+**This chart is not maintained by the Librespeed project and any issues with the chart should be raised [here](https://github.com/billimek/billimek-charts/issues/new)**
+
+## TL;DR;
+
+```shell
+helm repo add billimek https://billimek.com/billimek-charts/
+helm install billimek/Librespeed
+```
+
+## Introduction
+
+This code is adopted from the [Linuxserver Librespeed docker image](https://hub.docker.com/r/linuxserver/librespeed) which runs the [Librespeed application](https://github.com/librespeed/speedtest)
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```shell
+helm install --name my-release billimek/Librespeed
```

Comment:
This is really minor, but can we make this instance lowercase (e.g. `librespeed`)